seurat-find-transfer-anchor.R

```diff
@@ -0,0 +1,220 @@
+#!/usr/bin/env Rscript
+
+suppressPackageStartupMessages(require(optparse))
+suppressPackageStartupMessages(require(workflowscriptscommon))
+
+#set parsing options
+
+option_list = list(
+ 
+  
+   make_option(
+    c("--query-file"),
+    action = "store",
+    default = NA,
+    type = 'character',
+    help = "place to find query ",
```

Comment:
Try to match to the wrapped function's help text, so in this case ?FindTransferAnchors says 'Seurat object to use as the reference', so you could use 'File containing Seurat object to use as the reference'